src/App.js

```diff
@@ -1,36 +1,69 @@
-import React,{useState,useEffect} from 'react';
+import React, { useState, useEffect } from 'react';
 import TodoList from './components/TodoList'
 import AddTodoForm from './components/AddTodoForm'
 
-function useSemiPersistentState(){
-  const[todoList,setTodoList] = useState(localStorage.getItem('savedTodoList')? JSON.parse(localStorage.getItem('savedTodoList')):[])
-     
-  useEffect(() => {
-    window.localStorage.setItem('savedTodoList',todoList)
-     localStorage.setItem('savedTodoList',JSON.stringify(todoList))
-  },[todoList] )
-   return [todoList,setTodoList] 
-}
 
 function App() {
-  const[todoList,setTodoList] = useSemiPersistentState([]); 
-  const addTodo =(newTodo) =>{
-    setTodoList([...todoList,newTodo]) 
-  }  
-  function removeTodo(id){
-    const newArr = todoList.filter(function(item) {
-      return item.id !== id 
-    })
-      setTodoList(newArr)
+
+  const [todoList, setTodoList] = useState([])
+  const [isLoading, setIsLoading] = useState(true)
+
+  console.log(process.env)
+  console.log(process.env.REACT_APP_AIRTABLE_API_KEY)
+  useEffect(() => {
+    fetch(`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`,
+      {
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`
+
+        }
+      })
+      .then((response) => response.json())
+      .then((data) => {
+        setTodoList(data.records)
+        setIsLoading(false)
+      })
+
+  }, [])
+
+  useEffect(() => {
+    if (isLoading === false) {
+      window.localStorage.setItem('savedTodoList', todoList)
+      localStorage.setItem('savedTodoList', JSON.stringify(todoList))
+    }
+  }, [isLoading, todoList])
+  const addTodo = (newTodo) => {
     
+      fetch(`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`,
+           {
+             method:'POST',
+             headers:{
+              Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+              'Content-type': 'application/json'
+             },
+             body: JSON.stringify(newTodo)
+             
+           }).then(response =>response.json())
```

Comment:
Add a space between `=>` and `response.json`, and also pay attention to indentation. The code in this fetch request is correct, but the indents are not quite correct, so it is a bit difficult to understand.